cypress/e2e/tests/migration/applicationinventory/applications/language_discovery.test.ts

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    deleteByList,
+    getRandomApplicationData,
+    sidedrawerTab,
+} from "../../../../../utils/utils";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+import { SEC } from "../../../../types/constants";
+import { labelTagText } from "../../../../views/applicationinventory.view";
+
+let applicationList: Application[];
+
+describe(["@tier2"], "Test if application language discovered and tagged correctly", () => {
```

Comment:
Test if application language **is** discovered and tagged correctly

Reply:
done